assembly/src/ast/procedure/name.rs

```diff
@@ -317,13 +317,14 @@ impl FromStr for ProcedureName {
                     '_' | '$' => false,
                     _ => true,
                 }) {
-                    Err(IdentError::InvalidChars)
+                    // Err(IdentError::InvalidChars { ident: s.into() })
+                    panic!("IdentError::InvalidChars")
```

Comment:
I think you forgot to uncomment the `Err` here right? i.e. we shouldn't be panicking, and returning an error instead

Reply:
@greenhat - let's fix this issue and merge. This way, we can include this PR into the next release of `miden-vm` (which I'm hoping to do on Monday).

Reply:
Thanks for catching this! Fixed. 
Sorry, I should've picked it up earlier. I botched my email filter for github notifications and missed it.